console/HassClient.Performance.Tests/Program.cs

```diff
@@ -68,13 +74,11 @@ private static async Task ConnectToHomeAssistant(string ip, short port, bool eve
             {
                 Console.WriteLine($"Number of states: {client.States.Count}");
             }
-
-            var x = await client.GetState("sensor.temp_outside");
-
-            var services = await client.GetServices();
-
-            var config = await client.GetConfig();
-            System.Console.WriteLine(config.State);
+            
+            var data = new EventData {Temperature=4.3};
+            await client.TriggerWebhook("my-super-secret-id", data);
+            // await client.TriggerWebhook("my-super-secret-id", new {temperature = 4.1});
+           

```

Comment:
Tests should include no data

Reply:
This is a program I use to test real integration. Not actual "tests" :) Just some manual testing to show that it "actually" works.